File: src/main/java/com/thunderbear06/item/ItemRegistry.java
package com.thunderbear06.item;

import com.thunderbear06.CCAndroids;
import net.minecraft.world.item.Item;
import net.minecraft.world.item.SpawnEggItem;
import net.neoforged.neoforge.registries.DeferredItem;

public final class ItemRegistry {
    public static final DeferredItem<Item> WRENCH = CCAndroids.WRENCH;
    public static final DeferredItem<Item> COMPONENTS = CCAndroids.COMPONENTS;
    public static final DeferredItem<Item> ANDROID_CPU = CCAndroids.ANDROID_CPU;
    public static final DeferredItem<Item> REDSTONE_REACTOR = CCAndroids.REDSTONE_REACTOR;
    public static final DeferredItem<Item> ANDROID_FRAME = CCAndroids.ANDROID_FRAME;
    public static final DeferredItem<SpawnEggItem> ANDROID_SPAWN_EGG = CCAndroids.ANDROID_SPAWN;
    public static final DeferredItem<SpawnEggItem> ANDROID_ADVANCED_SPAWN_EGG = CCAndroids.ADVANCED_ANDROID_SPAWN;
    public static final DeferredItem<SpawnEggItem> ANDROID_COMMAND_SPAWN_EGG = CCAndroids.COMMAND_ANDROID_SPAWN;
    public static final DeferredItem<SpawnEggItem> ANDROID_ROGUE_SPAWN_EGG = CCAndroids.ROGUE_ANDROID_SPAWN;

    private ItemRegistry() {
    }
}

File: src/main/kotlin/com/thunderbear06/item/ItemRegistry.kt
package com.thunderbear06.item

import com.thunderbear06.CCAndroids
import net.minecraft.world.item.Item
import net.minecraft.world.item.SpawnEggItem
import net.neoforged.neoforge.registries.DeferredItem

object ItemRegistry {
    @JvmField val WRENCH: DeferredItem<Item> = CCAndroids.WRENCH
    @JvmField val COMPONENTS: DeferredItem<Item> = CCAndroids.COMPONENTS
    @JvmField val ANDROID_CPU: DeferredItem<Item> = CCAndroids.ANDROID_CPU
    @JvmField val REDSTONE_REACTOR: DeferredItem<Item> = CCAndroids.REDSTONE_REACTOR
    @JvmField val ANDROID_FRAME: DeferredItem<Item> = CCAndroids.ANDROID_FRAME
    @JvmField val ANDROID_SPAWN_EGG: DeferredItem<SpawnEggItem> = CCAndroids.ANDROID_SPAWN
    @JvmField val ANDROID_ADVANCED_SPAWN_EGG: DeferredItem<SpawnEggItem> = CCAndroids.ADVANCED_ANDROID_SPAWN
    @JvmField val ANDROID_COMMAND_SPAWN_EGG: DeferredItem<SpawnEggItem> = CCAndroids.COMMAND_ANDROID_SPAWN
    @JvmField val ANDROID_ROGUE_SPAWN_EGG: DeferredItem<SpawnEggItem> = CCAndroids.ROGUE_ANDROID_SPAWN
}
